cmd/celotool/send-tx.go

```diff
@@ -0,0 +1,139 @@
+// Copyright 2024 The celo Authors
+// This file is part of go-ethereum.
+//
+// go-ethereum is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// go-ethereum is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with go-ethereum. If not, see <http://www.gnu.org/licenses/>.
+
+package main
+
+import (
+	"context"
+	"fmt"
+	"math/big"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/core/types"
+	"github.com/ethereum/go-ethereum/crypto"
+	"github.com/ethereum/go-ethereum/ethclient"
+	"github.com/urfave/cli/v2"
+)
+
+var (
+	rpcUrlFlag = &cli.StringFlag{
+		Name:    "rpc-url",
+		Aliases: []string{"r"},
+		Usage:   "The rpc endpoint",
+	}
+	privKeyFlag = &cli.StringFlag{
+		Name:  "private-key",
+		Usage: "Use the provided private key",
+	}
+)
+
+var commandSend = &cli.Command{
+	Name:      "send",
+	Usage:     "send celo tx (cip-64)",
+	ArgsUsage: "[to] [feeCurrency] [value]",
```

Comment:
What about max fee in fee currency? Also it would be good to describe the args i.e what format and unit are they? Maybe a short example of good args

Reply:
So far it only supports CIP-64, not CIP-66.

Reply:
Added some basic description.